student.rb

```diff
@@ -0,0 +1,18 @@
+# student.rb
+
+require_relative 'person'
+
+# Represents a student with an additional classroom.
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(id, age = 18, name = 'Unknown', parent_permission: true, classroom: 'Ruby')
+    # Call the parent's constructor using super
+    super(id, age, name, parent_permission)
```

Comment:
- [ ] age and classroom cannot be optional, they are required.

- [ ] The parent_permission should also be passed correctly in line 11.

  ```
  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
    super(id, age, name, parent_permission: parent_permission)
  ```

- [ ] **[OPTIONAL]** If you took the optional suggestion of removing the `id` above, then you need to get rid of the `id` here too.